index.rst

```diff
@@ -1,61 +1,419 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+Introduction
+============
 
-  Sections
-  ========
+   There are only two hard things in Computer Science: cache invalidation and
+   naming things.
 
-  and
+   — Phil Karlton
 
-  Subsections
-  -----------
+We assert that hostnames are primarily for **human** convenience and should be
+easy to remember, easy to predict, and concise.  This should be the primary
+guiding principle when developing naming heuristic(s).
 
-  and
+The current naming scheme being implemented at the summit was widely disliked
+during discussion at the `Puppeton Tucson July 2019 <https://confluence.lsstcorp.org/display/LSSTCOM/Agenda+-+Puppeton+Tucson+July+2019>`_
+as being overly complex and difficult to understand.  During that event, it was
+agreed upon to `decouple the puppet role from the hostname
+<https://ittn-001.lsst.io/#decouple-node-fqdn-from-hierarchy-layers>`_, this
+change opens the door to revising the hos tame and domain name strategy.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+Implementing a reformed DNS strategy is on the critical path for rolling out a
+`Deployment Platform <https://ittn-002.lsst.io/>`_ at the Summit and Base to
+support commissioning activities.  A practical and rapidly implementable
+solution is urgently needed, even if it is not the complete solution that will
+be used during observatory operations.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Goals
+=====
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+* Support immediate development and commissioning needs
```

Comment:
Either make all of these imperative verbs (e.g. "support") or all of them nouns ("implementation").  Don't mix 'em.